### PR TITLE
CPO-11 added config for new micro service case-payment-order-api

### PR DESCRIPTION
**Pull request by smathangi**
… case-payment-order-api

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by nathan-clark — DISMISSED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -63,6 +63,11 @@ k8s/**/common/ccd/case-management-web.yaml @hmcts/cdm
 k8s/namespaces/aac/ @hmcts/cdm
 k8s/**/common/aac/identity.yaml @hmcts/cdm
 
+##CPO - CDM
+k8s/namespaces/aac/ @hmcts/cdm
+k8s/aat/common/aac/identity.yaml @hmcts/cdm
+k8s/demo/common/aac/identity.yaml @hmcts/cdm
```
Comment (danlysiak) [inline CODEOWNERS:69]:
```suggestion
k8s/namespaces/cpo/ @hmcts/cdm
k8s/aat/common/cpo/identity.yaml @hmcts/cdm
k8s/demo/common/cpo/identity.yaml @hmcts/cdm
```

File: bin/base-ns-id-gen.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 SUB_MAP=( "aat=DCD-CNP-DEV"
-  "perftest=DCD-CNP-QA"
+  "demo=DCD-CNP-DEV"
```
Comment (danlysiak) [inline bin/base-ns-id-gen.sh:4]: Are we meant to be committing these changes?

File: k8s/namespaces/cpo/namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cpo
+  labels:
+    slackChannel: cpo
```
Comment (danlysiak) [inline k8s/namespaces/cpo/namespace.yaml:6]:
```suggestion
    slackChannel: ccd-master-builds
```
Maybe?

File: k8s/namespaces/cpo/kustomization.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: cpo
+bases:
+- namespace.yaml
+# Warning : Adding a file here, adds to all environments to which you add your kustomization.
```
Comment (danlysiak) [inline k8s/namespaces/cpo/kustomization.yaml:6]: Do we need to add in the cpo base too?

Comment (danlysiak) [reply k8s/namespaces/cpo/kustomization.yaml:6]: Sorry, ignore.. see we're just adding to select envs atm

File: bin/base-ns-id-gen.sh (hunk 2 above)
Comment (smathangi) [reply bin/base-ns-id-gen.sh:4]: local change 😄

**Review by danlysiak — APPROVED** [review]

**Review by luigibk — APPROVED** [review]

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**